pkg/api/v1/auth.go

```diff
@@ -30,3 +42,57 @@ func (r *LoginRequest) Validate() (err error) {
 
 	return err
 }
+
+// Redirect returns the next URL to redirect to after a successful login.
+// If no next URL is provided, it defaults to the root path ("/").
+// TODO: should we allow this to be configurable?
+func (r *LoginRequest) Redirect() string {
+	// Check if we have a next URL to redirect to with the request.
+	if r.Next != "" {
+		return r.Next
+	}
+
+	// If all else fails, redirect to the root path.
+	return "/"
+}
+
+func (r *AuthenticateRequest) Validate() (err error) {
+	r.ClientID = strings.TrimSpace(r.ClientID)
+	if r.ClientID == "" {
+		err = ValidationError(err, MissingField("client_id"))
+	}
+
+	r.ClientSecret = strings.TrimSpace(r.ClientSecret)
+	if r.ClientSecret == "" {
+		err = ValidationError(err, MissingField("client_secret"))
+	}
+	return err
+}
+
+func (r *AuthenticateRequest) Redirect() string {
```

Comment:
Does this make sense or were we planning on handling redirects differently for these endpoints?

Reply:
I actually just removed these and replaced the redirect selection with configurable endpoints:

https://app.shortcut.com/rotational/story/33547/configure-login-and-logout-urls

So we're doing this now:

https://github.com/rotationalio/quarterdeck/blob/main/pkg/server/auth.go#L147-L151